catkit2/services/thorlabs_tsp01/thorlabs_tsp01.py

```diff
@@ -16,10 +16,12 @@ def __init__(self):
         self.serial_number = self.config['serial_number']
         self.num_averaging = self.config.get('averaging', 1)
         self.interval = self.config.get('interval', 10)
+        self.read_out_header_1 = self.config.get('read_out_header_1', True)
+        self.read_out_header_2 = self.config.get('read_out_header_2', True)
 
         self.temperature_internal = self.make_data_stream('temperature_internal', 'float64', [1], 20)
-        self.temperature_header_1 = self.make_data_stream('temperature_header_1', 'float64', [1], 20)
-        self.temperature_header_2 = self.make_data_stream('temperature_header_2', 'float64', [1], 20)
+        self.temperature_header_1 = self.make_data_stream('temperature_header_1', 'float64', [1], 20) if self.read_out_header_1 else None
```

Comment:
Make this and the next line a normal if-statement instead of a terniary. Our variable doesn't need `None` as a default and can just not exist.

```python
if self.read_out_header_1:
    self.make_data_stream('temperature_header_1', 'float64', [1], 20)
```